File: respostas/Java/Beginner/1017/Main.java
import java.util.Scanner; 

public class Main {
 
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        int time = scan.nextInt(), speed = scan.nextInt();
        
        double distance = speed * time, liters = distance / 12;
        
        System.out.printf("%.3f\n", liters);
        
        scan.close();
    }
 
}

File: respostas/C/Beginner/1017/main.c
#include <stdio.h>

int main(void) {

    //12km/L

    int hours, speed, distance;
    scanf("%d%d", &hours, &speed);

    distance = hours * speed;

    float liters = (float)distance/12;
    printf("%.3f\n", liters);
    return 0;
}